modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -13,8 +13,17 @@ import (
 	"github.com/cosmos/ibc-go/v8/modules/apps/29-fee/types"
 	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+	ibcmock "github.com/cosmos/ibc-go/v8/testing/mock"
 )
 
+func enableFeeOnChannel(path *ibctesting.Path) {
```

Comment:
Sure, will do!

Reply:
(getting back to this, I think it might need a bit more work, as `fee_test.go` is a test package that can not be imported directly. It would need to go in a shared package between those. It can be made to work but I'll need to think a bit about it :)

Reply:
ah, yes, I see. Maybe we can add an internal package?

Reply:
What about adding them [here](https://github.com/cosmos/ibc-go/blob/c374041dd36205fe081f4d61443f1505ab665737/testing/utils.go)? 

It seems to already contain enough utility functions for testing and my understanding is that it would be fine there. I pushed this change but happy to revert and create a new package/put it somewhere else!

Reply:
the [utils function suggestion](https://github.com/cosmos/ibc-go/blob/c374041dd36205fe081f4d61443f1505ab665737/testing/utils.go) makes sense to me!